package.json

```diff
@@ -43,7 +43,7 @@
     "mongoose": "^6.2.0",
     "morgan": "^1.10.0",
     "os": "^0.1.2",
-    "passport": "^0.4.1",
+    "passport": "^0.5.2",
```

Comment:
Breaking changes introduced in passport@0.5.x. (Middleware no longer served via http incoming request).